.travis.yml

```diff
@@ -13,5 +13,13 @@ before_install:
 install:
   - pip install tox-travis
 
-script: tox
+script: tox -r
```

Comment:
I guess -r is run when it realizes there isn't a tox environment. So it probably wasn't necessary. Did it out of habit.